lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
Also, even if we need to keep the existence check on the filesystem, you should simply use `File.exists?(pname)`, which will simply return `false` if the file isn't on the filesystem. That way you don't need the `begin/rescue/return` flow!

(Sorry for the multiple comments)